packages/aws-cdk/lib/context-providers/ami.ts

```diff
@@ -8,7 +8,7 @@ import { ContextProviderPlugin } from './provider';
  * Plugin to search AMIs for the current account
  */
 export class AmiContextProviderPlugin implements ContextProviderPlugin {
-  constructor(private readonly aws: SdkProvider) {
+  constructor(private readonly aws: SdkProvider, private readonly assumeRoleArn?: string) {
```

Comment:
Additional constructor parameter no longer necessary if the role ARNs are already inside `XxxContextQuery`.